packages/homepage/src/components/PostElements.js

```diff
@@ -0,0 +1,40 @@
+import styled from 'styled-components';
+
+export const Container = styled.div`
+  color: rgba(255, 255, 255, 0.9);
+  margin-bottom: 1rem;
+`;
+
+export const Title = styled.h2`
+  font-family: 'Poppins';
+  font-weight: 600;
+  font-size: 30px;
+  line-height: 1.5;
+
+  color: #f2f2f2;
+`;
+
+export const PostDate = styled.span`
+  font-family: 'Poppins';
```

Comment:
I know that I've been doing this wrong as well, but would be better to have this as a fallback.

```suggestion
  font-family: 'Poppins', sans-serif;
```